communities/views.py

```diff
@@ -1,3 +1,90 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from .models         import Community
+from creators.models import Creator
+from users.utils     import login_decorator
+
+class CommunityView(View):
+    def get(self, request, product_id):
+        communities   = Community.objects.filter(product_id=product_id)
+        creator       = Creator.objects.get(product_id=product_id)
+        user_id       = creator.user.id
+        announcements = Community.objects.filter(product_id=product_id, user_id=user_id)
+
+        reviews = [
+            {
+                'user_id'    : community.user.id,
+                'name'       : community.user.name,
+                'product_id' : community.product.id,
+                'content'    : community.content,
+                'id'         : community.id,
+                'post_id'    : community.parent_review.id
+            } for community in communities if community.parent_review.id
+        ]
+
+        announcements = [
+            {
+                'creator'    : announcement.user.creator_set.id,
+                'product_id' : announcement.product_id,
+                'content'    : announcement.content
+            } for announcement in announcements
+        ]
+
+        communities = [
+            {
+                'name'      : community.user.id,
+                'product_id': community.product.id,
+                'content'   : community.content,
+                'id'        : community.id
+
+            } for community in communities if not community.parent_review.id
+        ]
+
+        return JsonResponse({'postings':communities, 'announcements':announcements, 'comments':reviews}, status=200)
+
+    @login_decorator
+    def post(self, request, product_id):
+        data = json.loads(request.body)
+
+        Community.objects.create(
+            user_id          = request.user,
```

Comment:
동작하는데는 문제가 없더라도 좀 더 명확하게 적어주시는게 좋을 것 같아요!
```suggestion
            user          = request.user,
```